requirements.txt

```diff
@@ -1,5 +1,6 @@
 gitpython
 pyyaml
+conda ==4.1.12
 conda-build-all
 conda-build ==2.0.6
-requests !=2.12.0
+requests ==2.11.1 
```

Comment:
Perhaps we could unpin `conda` and get `conda-gitenv` to work with both versions ...